public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -85,7 +85,7 @@ export default class InfluxDatasource {
     // replace templated variables
     allQueries = this.templateSrv.replace(allQueries, scopedVars);
 
-    return this._seriesQuery(allQueries).then((data): any => {
+    return this._seriesQuery(allQueries, targets[0]).then((data): any => {
```

Comment:
this seems like it will cause issues, using the first query only for the options.